Makefile

```diff
@@ -53,12 +53,13 @@ lint:
 
 .PHONY: build
 build: deps
-	$(GO) install -x -ldflags "$(GOBUILD_LDFLAGS)" $(ALL_PACKAGES)
+	GOOS=linux $(GO) build -o gfmrun_linux_$(VERSION_VALUE)  -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
+	GOOS=darwin $(GO) build -o gfmrun_darwin_$(VERSION_VALUE) -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
+	GOOS=windows $(GO) build -o gfmrun_windows_$(VERSION_VALUE).exe  -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
```

Comment:
I think we should change to dashes from underscores, if you think how golang does it itself for it's own packages, it's usually (app)-(goos)-(goarch)(optional ext for windows). gfmrun_win.exe doesn't tell me if its running on my microsoft surface that uses an arm processor.

Reply:
Sure. Give me a moment.